lib/v2/bilibili/radar.js

```diff
@@ -54,6 +54,24 @@ module.exports = {
                 source: '/:uid',
                 target: '/bilibili/user/video-all/:uid',
             },
+            {
+                title: 'UP 主当前合集',
```

Comment:
```suggestion
                title: 'UP 主频道的合集',
```
`title` should match the H3 heading